tools/setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#usage: setup.sh [path/to/tizen-studio] [path/to/iotjs]
+
+[ "$#" -ne 2 ] && { echo "Usage: $0 <tizen-studio-path> <iotjs-path>"; exit 1; }
+
+TIZEN_STUDIO=$1
+IOTJS=$2
+
+if [ -e $TIZEN_STUDIO/platforms/tizen-4.0/iot-headless/ ]; then
+    exit 0
+fi
+if [ ! -e $TIZEN_STUDIO/platforms/tizen-4.0/iot-headless/ ]; then
```

Comment:
Unnecessary condition. (There is an early return before this condition, so this `if` statement can be removed.)